src/__init__.py

```diff
@@ -0,0 +1,3 @@
+import logging
+
+logging.basicConfig(level=logging.WARNING)
```

Comment:
The level should be collected from `settings.py` or similar. Although at the moment this would have little to no benefit. I'd just ignore this.

Reply:
Done 👍